book3/13-web.mkd

```diff
@@ -253,152 +253,153 @@ crea un conjunto de *servicios* disponibles para que los usen otras aplicaciones
+realizar una reserva de un hotel usando el sitio web de una aerolínea, ésta utiliza
+otro servicio web en los sistemas de los hoteles para realizar la reserva real. Y
+cuando llega el momento de cargar en tu tarjeta de crédito el importe de la transacción completa, hay todavía otros equipos diferentes involucrados en el proceso.
+
 
 ![Service-oriented architecture](height=3.0in@../images/soa)
 
-A Service-oriented architecture has many advantages, including: (1) we
-always maintain only one copy of data (this is particularly important
-for things like hotel reservations where we do not want to over-commit)
-and (2) the owners of the data can set the rules about the use of their
-data. With these advantages, an SOA system must be carefully designed to
-have good performance and meet the user's needs.
+Una Arquitectura Orientada a Servicios tiene muchas ventajas, que incluyen: (1)
+siempre se mantiene una única copia de los datos (lo cual resulta particularmente
+importante en ciertas áreas como las reservas hoteleras, donde no queremos duplicar
+excesivamente la información) y (2) los propietarios de los datos pueden imponer
+reglas acerca del uso de esos datos. Con estas ventajas, un sistema SOA debe ser
+diseñado cuidadosamente para tener buen rendimiento y satisfacer las necesidades
+de los usuarios.
+
 
-When an application makes a set of services in its API available over
-the web, we call these *web services*.
+Cuando una aplicación ofrece un conjunto de servicios en su API disponibles a
+través de la web, los llamamos *servicios web*.
 
-Security and API usage
+Seguridad y uso de APIs
 ----------------------
 
 \index{OAuth}
 \index{API!key}
 
-It is quite common that you need an API key to make use of a
-vendor's API. The general idea is that they want to know who is using
-their services and how much each user is using. Perhaps they have free
-and pay tiers of their services or have a policy that limits the number
-of requests that a single individual can make during a particular time
-period.
-
-Sometimes once you get your API key, you simply include the key as part
-of POST data or perhaps as a parameter on the URL when calling the API.
-
-Other times, the vendor wants increased assurance of the source of the
-requests and so they expect you to send cryptographically signed
-messages using shared keys and secrets. A very common technology that is
-used to sign requests over the Internet is called
-*OAuth*. You can read more about the OAuth protocol at
-[www.oauth.net](http://www.oauth.net).
-
-Thankfully there are a number of convenient
-and free OAuth libraries so you can avoid writing an OAuth
-implementation from scratch by reading the specification. These
-libraries are of varying complexity and have varying degrees of
-richness. The OAuth web site has information about various OAuth
-libraries.
+Resulta bastante frecuente que se necesite algún tipo de “clave API” para hacer
```

Comment:
Not sure if the double quotes will break the book build. If you can change them to the "english" format, that would be better (from “clave API” to "clave API".

Reply:
efectuados todos los cambios mencionados en estos comentarios.